pkg/insights/resyncer.go

```diff
@@ -473,17 +479,17 @@ func (r *resyncer) createOrUpdateServiceInsight(
 			case mesh_proto.Dataplane_Networking_Gateway_DELEGATED:
 				svcType = mesh_proto.ServiceInsight_Service_gateway_delegated
 			}
-			populateInsight(svcType, insight, gw.GetTags()[mesh_proto.ServiceTag], status, backend, "")
+			populateInsight(svcType, insight, gw.GetTags()[mesh_proto.ServiceTag], status, backend, "", core_mesh.ProtocolHTTP)
```

Comment:
I don't think that's right.

Reply:
I am not fully sure how to handle this for the gateway. Gateway can have more than one listener with different protocols. Maybe I could just take all protocols from listeners and based on this call common protocol